drivers/sensor/bh1750/Kconfig

```diff
@@ -0,0 +1,50 @@
+# BH1750 ambient light sensor configuration options
+
+# Copyright (c) 2023 Sngular People SL.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+DT_COMPAT_ROHM_BH1750 := rohm,bh1750
+
+menuconfig BH1750   
+    bool "BH1750 ambient light sensor"
+    default y
+    depends on DT_HAS_ROHM_BH1750_ENABLED
+    select I2C
+    help 
+        BH1750 ambient light sensor I2C-based
+ 
+if BH1750
+
+choice BH1750_DRIVER_MODE
+    prompt "BH1750 driver mode"
+    default BH1750_MODE_CONTINOUSLY
+    help 
+        Specifies the measurement mode used by the driver.
+
+config BH1750_MODE_CONTINOUSLY
+    bool "A new measurement is ready continuously. the period time depends on resolution mode"
+
+config BH1750_MODE_ONE_TIME
+    bool "Perform only one new measurement. It is automatically set to Power Down mode after measurement."
+
+endchoice
+
+choice BH1750_RESOLUTION
+    prompt "BH1750 driver resolution"
+    default BH1750_H_RESOLUTION_MODE
+    help
+        Specifies the resolution mode used by the driver
+
+config BH1750_H_RESOLUTION_MODE
+    bool "1lx resolution. Measurement Time is typically 120ms."
+
+config BH1750_H_RESOLUTION_MODE2
+    bool "0.5lx resolution. Measurement Time is typically 120ms."
+
+config BH1750_L_RESOLUTION_MODE
+    bool "4lx resolution. Measurement Time is typically 16ms."
+
+endchoice
```

Comment:
Please move to devicetree properties

Reply:
Just to be sure I do correctly. 
You mean to move to `dts/bindigns/sensor/rohm,bh1750.yaml` , don't you?
Thus, the sensor will not require any Kconfig variable and all settings would be configured by the device tree, right?
And just for curiosity, is it the new tendency of Zephyr development? 
Thank you in advance!

Reply:
Ok, thank you very much for the clarification. 
Done